src/embed.ts

```diff
@@ -336,27 +337,45 @@ class Torus {
     return response;
   }
 
-  async sendTransaction(transaction: Transaction): Promise<string> {
+  async sendTransaction(transaction: TransactionOrVersionedTransaction): Promise<string> {
+    const transactionVersion = findTransactionVersion(transaction);
+    const message =
+      transactionVersion === "0"
```

Comment:
should not check for zero, as if future version introduced, embed need to be updated